allensdk/brain_observatory/behavior/running_processing.py

```diff
@@ -14,40 +16,378 @@ def calc_deriv(x, time):
     return dxdt
 
 
-def deg_to_dist(speed_deg_per_s):
-    '''
-    takes speed in degrees per second
-    converts to radians
-    multiplies by radius (in cm) to get linear speed in cm/s
-    '''
+def _angular_change(summed_voltage: np.ndarray,
+                    vmax: Union[np.ndarray, float]) -> np.ndarray:
+    """
+    Compute the change in degrees in radians at each point from the
+    summed voltage encoder data.
+
+    Parameters
+    ----------
+    summed_voltage: 1d np.ndarray
+        The "unwrapped" voltage signal from the encoder, cumulatively
+        summed. See `_unwrap_voltage_signal`.
+    vmax: 1d np.ndarray or float
+        Either a constant float, or a 1d array (typically constant)
+        of values. These values represent the theoretical max voltage
+        value of the encoder. If an array, needs to be the same length
+        as the summed_voltage array.
+    Returns
+    -------
+    np.ndarray
+        1d array of change in degrees in radians from each point
+    """
+    delta_theta = np.diff(summed_voltage, prepend=np.nan) / vmax * 2 * np.pi
+    return delta_theta
+
+
+def _shift(
+        arr: Iterable,
+        periods: int = 1,
```

Comment:
periods is an unusual name for this. Period implies a time or distance scale for a signal, i.e. in a 1/frequency sense. Why not ust mirror the language from `numpy.roll`?
Also, periods not covered by docstring.

Reply:
Will update docstring. Doug's algorithm used a pandas function to do this that had slightly different behavior than numpy. I forget exactly what it's called, but the pandas function had `periods` as the arg which is why I used it. I will keep the name because of that.